schemas/components/schemas/WebhookRule.yaml

```diff
@@ -0,0 +1,14 @@
+x-internal: true
```

Comment:
@erayaydin Is it expected that internal schemas are synced to OpenAPI repo?  I don't think so